.eslintrc.cjs

```diff
@@ -75,21 +76,35 @@ module.exports = {
       },
     },
     {
-      files: ['e2e/**/*.js'],
+      files: ['src/**/*.cjs'],
```

Comment:
```suggestion
      files: ['src/**/*.{js,mjs,cjs}'],
```

Reply:
`no-restircted-modules` only applies to commonjs. for esm there is `no-restricted-imports`